bucket/meld.json

```
@@ -20,6 +21,6 @@
         "url": "http://meldmerge.org/"
     },
     "autoupdate": {
-        "url": "https://download.gnome.org/binaries/win32/meld/$majorVersion.$minorVersion/Meld-$version-win32.msi"
+        "url": "https://download.gnome.org/binaries/win32/meld/$majorVersion.$minorVersion/Meld-$version-mingw.msi"
```

Add hash extraction: (<https://download.gnome.org/binaries/win32/meld/3.20/Meld-3.20.0-mingw.msi.mirrorlist>)

```suggestion
        "url": "https://download.gnome.org/binaries/win32/meld/$majorVersion.$minorVersion/Meld-$version-mingw.msi",
        "hash": {
            "url": "$url.sha256"
        }
```